### ISSUE TITLE
Automatically publish nightly versions?

### ISSUE THREAD

**Issue by vilic** [project-chip/matter.js#364]
Just encountered another issue (re-announce after subscription canceled by peer) and found already fixed in the code base but not yet released.

Maybe publish nightly versions automatically, consider the code base is now somewhat unstable?

**Comment by Apollon77** [comment]
We are already on that, as soon as I get some Github settings done by people with the matching rights we will have that ... see https://github.com/orgs/project-chip/projects/11?pane=issue&itemId=36578941 (will close because more a todo then an issue). Also 0.6 will come soon once the big MDNS PR is merged (ideally also automated) :-)

**Comment by Apollon77** [comment]
PS: "Somewhat unstable" is not really correct ;-) Honestly ... that reannounce is a "reality driven change" we added that are not covered by any specs or such, so in fact it should be the controllers to try from themself when such ancase happened and not rely on a re-announce ... but yes with this "feature" it works better. With the MDNS branch there willbe more improvements

**Comment by Apollon77** [comment]
docs will be added in https://github.com/project-chip/matter.js/pull/368/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5

**Comment by vilic** [comment]
@Apollon77 Haha thanks. By "unstable" I mean actively evolving.

I just started learning Matter (by getting hands on Matter.js), so pardon me as I am certainly not clear about many details. 😃
